CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# dbt_linkedin v1.0.0
+[PR #50](https://github.com/fivetran/dbt_linkedin/pull/50) includes the following updates:
+
+## Breaking Changes
+> A `--full-refresh` is required after upgrading to this release.
+
+### Source Package Consolidation
+- Consolidated `dbt_linkedin_source` into this package.
+- All functionality from the source package has been merged into this transformation package. Be sure to:
+  - Remove `fivetran/linkedin_source` from `packages.yml`
+  - Delete any `source:` overrides that reference the source package
```

Comment:
If they are still on a dbt version that supports overrides, they can update the `overrides:` config to be `overrides: linkedin` instead of `overrides: linkedin_source`